MDB2/Driver/pgsql.php

```diff
@@ -215,7 +215,10 @@ public function escape($text, $escape_wildcards = false)
         if (MDB2::isError($connection)) {
             return $connection;
         }
-        if (is_resource($connection) && version_compare(PHP_VERSION, '5.2.0RC5', '>=')) {
+
+        if ((is_a($connection, 'PgSql\Connection') || is_resource($connection))
+            && version_compare(PHP_VERSION, '5.2.0RC5', '>=')
```

Comment:
I'm quite tempted to remove the PHP 5.2 support. I think we already declare the package requires 5.3+